src/client/sandbox/code-instrumentation/methods.js

```diff
@@ -1,12 +1,46 @@
 import SandboxBase from '../base';
 import { isNullOrUndefined, inaccessibleTypeToStr, isWindow, isDocument } from '../../utils/types';
 import { DOCUMENT_WRITE_BEGIN_PARAM, DOCUMENT_WRITE_END_PARAM, CALL_METHOD_METH_NAME } from '../../../processing/js';
+import { FOCUS_PSEUDO_CLASS_ATTR } from '../../../const';
+import { isDomElement } from '../../utils/dom';
```

Comment:
It's quite fragile logic. What should go there? Should `isShadowUIElement` be in types? `utils/types` currently is the `god-util` for me. I'd preffer to have things separated dom types in `dom` utils, style types in `style` utils and so on. @churkin I don't get what `isDomElement` checks for? At first I thought it checks for node being an `HTMLElement`, but from code it doesn't seems to be a true.